### PR TITLE
removed underscore

### PR DESCRIPTION
**Pull request by axton21**

### CODE REVIEW

**Comment by axton21** [comment]
@DFurnes

File: app/views/users/partials/index-table.blade.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <table id="user-table">
 	<thead>
 		<tr class="row table-header">
-			<th class="table-cell">_id</th>
+			<th class="table-cell">Id</th>
```
Comment (DFurnes) [inline app/views/users/partials/index-table.blade.php]: Would it make more sense to rename this "User ID"?

**Comment by axton21** [comment]
sure...

**Comment by axton21** [comment]
@DFurnes

**Comment by DFurnes** [comment]
:boom: :shipit:

**Pull request merged**